src/sentry/incidents/logic.py

```diff
@@ -655,15 +667,39 @@ def validate_alert_rule_query(query):
     get_snuba_query_args(query)
 
 
-def create_snuba_subscription(dataset, query, aggregations, time_window, resolution):
+def create_snuba_subscription(project, dataset, query, aggregations, time_window, resolution):
     """
     Creates a subscription to a snuba query.
 
-    :param alert_rule: The alert rule to create the subscription for
+    :param project: The project we're applying the query to
+    :param dataset: The snuba dataset to query and aggregate over
+    :param query: An event search query that we can parse and convert into a
+    set of Snuba conditions
+    :param aggregations: A list of aggregations to calculate over the time
+    window
+    :param time_window: The time window to aggregate over
+    :param resolution: How often to receive updates/bucket size
     :return: A uuid representing the subscription id.
     """
-    # TODO: Implement
-    return uuid4()
+    # TODO: Might make sense to move this into snuba if we have wider use for
+    # it.
```

Comment:
Would this have a wrapper method in the `utils.snuba` module down the road?

Reply:
I think that makes sense.